mteb/tasks/BitextMining/multilingual/RuSciBenchBitexMining.py

```diff
@@ -0,0 +1,71 @@
+from __future__ import annotations
+
+import datasets
+
+from mteb.abstasks.AbsTaskBitextMining import AbsTaskBitextMining
+from mteb.abstasks.MultilingualTask import MultilingualTask
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+_LANGUAGES = {
+    "ru-en": ["rus-Cyrl", "eng-Latn"],
+    "en-ru": ["eng-Latn", "rus-Cyrl"],
+}
+
+
+_SPLITS = ["test"]
+
+
+class RuSciBenchBitexMining(AbsTaskBitextMining, MultilingualTask):
+    fast_loading = True
+    metadata = TaskMetadata(
+        name="RuSciBenchBitexMining",
+        dataset={
+            "path": "mlsa-iai-msu-lab/ru_sci_bench_bitext_mining",
+            "revision": "927d95897a168b79568e96591276b995ed1c4da8",
+        },
+        description="Find translation of a scientific article",
```

Comment:
I would love a more detailed description here. What scientific articles, how was the translation done etc.